fdbserver/ptxn/test/TestTLogServer.actor.cpp

```diff
@@ -1049,22 +849,24 @@ TEST_CASE("/fdbserver/ptxn/test/single_tlog_recovery") {
 	state ptxn::TLogGroupID targetGroup = pContext->storageTeamIDTLogGroupIDMapper[storageTeamID];
 	state std::unordered_map<ptxn::TLogGroupID, std::pair<IKeyValueStore*, IDiskQueue*>> dqs;
 	state int writtenTLogID = pContext->groupToLeaderId[pContext->storageTeamIDTLogGroupIDMapper[storageTeamID]];
-	PromiseStream<ptxn::InitializePtxnTLogRequest> initializeTLogRecover;
+	state PromiseStream<ptxn::InitializePtxnTLogRequest> initializeTLogRecover;
 
-	state IDiskQueue::location previousNextPushLocation = qs[targetGroup]->getNextPushLocation();
+	state IDiskQueue::location previousNextPushLocation = pContext->qs[targetGroup]->getNextPushLocation();
+	StringRef fileVersionedLogDataPrefix = "log2-"_sr;
+	StringRef fileLogDataPrefix = "log-"_sr;
+	std::string diskQueueFilePrefix = "logqueue-";
 
-	tLogInitializations.emplace_back();
-	tLogInitializations.back().isPrimary = true;
-	tLogInitializations.back().storeType = KeyValueStoreType::MEMORY;
-	tLogInitializations.back().tlogGroups = pContext->groupsPerTLog[writtenTLogID];
+	state ptxn::InitializePtxnTLogRequest req;
```

Comment:
Does `req` need to be a state variable (also for initializeTLogRecover above)? There is no `wait` between here and their last usage.

Reply:
good catch, removed